.github/workflows/check-pull-request.yaml

```
@@ -43,7 +43,7 @@ jobs:
     - uses: actions/checkout@df4cb1c069e1874edd31b4311f1884172cec0e10  # v6
       with:
         persist-credentials: false
-    - uses: ./.github/actions/setup-python
+    - uses: ./.github/actions/setup-dev
```

actionlint (1.7.12)

[error] description is required in metadata of "Setup development tools" action at "/home/jailuser/git/.github/actions/setup-dev/action.yaml"

(action)